src/components/BottomBar/SkyBrowser/SkyBrowserTabs.jsx

```diff
@@ -14,74 +14,165 @@ import {
   reloadPropertyTree,
 } from '../../../api/Actions';
 
-class SkyBrowserTabs extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      isShowingInfoButtons: [false, false, false, false, false],
-      showSettings: false,
-      messageCounter: 0
-    };
-    this.createTabs = this.createTabs.bind(this);
-    this.createImageList = this.createImageList.bind(this);
-    this.showTooltip = this.showTooltip.bind(this);
-    this.hideTooltip = this.hideTooltip.bind(this);
-    this.createButtons = this.createButtons.bind(this);
-    this.toggleShowSettings = this.toggleShowSettings.bind(this);
-    this.setImageLayerOrder = this.setImageLayerOrder.bind(this);
-    this.createTargetBrowserPair = this.createTargetBrowserPair.bind(this);
-    this.removeTargetBrowserPair = this.removeTargetBrowserPair.bind(this);
+function SkyBrowserTabs({ 
+  setCurrentTabHeight,
+  passMessageToWwt,
+  setWwtRatio,
+  activeImage,
+  currentBrowserColor,
+  selectImage,
+  maxHeight,
+  minHeight,
+  height,
+  setBorderRadius,
+  imageCollectionIsLoaded
+}) {
+  const [isShowingInfoButtons, setIsShowingInfoButtons] =
```

Comment:
the line breaking for this seems a bit odd to me. Maybe
```
const [isShowingInfoButtons, setIsShowingInfoButtons] = React.useState(
   [false, false, false, false, false]
);
```

Or something? Also, for me it is a bit difficult what is going on on this line. Why the array of falses? 

```